api/serverpb/json/client/server/logs_parameters.go

```diff
@@ -58,6 +59,12 @@ func NewLogsParamsWithHTTPClient(client *http.Client) *LogsParams {
    Typically these are written to a http.Request.
 */
 type LogsParams struct {
+	/* Pprof.
+
+	   Include performance profiling data,
+	*/
```

Comment:
Can we use one-line comment here?

Reply:
Well, this file is generated automaticly...

Reply:
Damn 😖